test/e2e/tests/simple-send.spec.js

```diff
@@ -14,14 +19,28 @@ describe('Simple send', function () {
         ganacheOptions: defaultGanacheOptions,
         title: this.test.fullTitle(),
       },
-      async ({ driver, ganacheServer }) => {
-        await logInWithBalanceValidation(driver, ganacheServer);
+      async ({ driver }) => {
+        // Process
+        await unlockWallet(driver, WALLET_PASSWORD);
 
-        await sendTransaction(
-          driver,
+        // Page Objects and Components
+        const confirmTxPage = new ConfirmTxPage(driver);
+        const homePage = new HomePage(driver);
+        const sendAmountPage = new SendAmountPage(driver);
+        const sendToPage = new SendToPage(driver);
+
+        // Actions and Assertions
+        await homePage.startSendFlow();
+        await sendToPage.addRecipient(
           '0x985c30949c92df7a0bd42e0f3e3d539ece98db24',
-          '1',
         );
+        await sendAmountPage.addAmount('1');
+        await sendAmountPage.goToNextScreen();
+        await confirmTxPage.confirmTx();
+        await homePage.goToActivityList();
```

Comment:
all the actions we perform in the test should be clear by their name. This can also be seen as documentation of flows. From this test we infer this flow:
1. login
2. start a send flow
3. add a recipient address
4. add a token amount
5. proceed to the next screen
6. confirm transaction
7. go to activity list
8. (in the assertion) check tx is there